feeds/fr.json

```diff
@@ -351,6 +351,30 @@
             "managed-by-script": true,
             "skip": false
         },
+        {
+            "name": "reseau-urbain-et-interurbain-dile-de-france-mobilites",
+            "type": "url",
+            "url": "http://gtfsidfm.clarifygdps.com/gtfs-rt-trips-idfm",
+            "license": {
+                "url": "https://transport.data.gouv.fr/datasets/reseau-urbain-et-interurbain-dile-de-france-mobilites",
+                "spdx-identifier": null
+            },
+            "spec": "gtfs-rt",
+            "x-data-gov-fr-res-id": 80921,
```

Comment:
If this is from gouv.fr, it should already be part of this file.
Otherwise, please remove the id. It is only used for automatic updates to this file from gouv.fr